source/Refitting/src/ExtrToSIT.cc

```diff
@@ -217,7 +221,15 @@ void ExtrToSIT::init() {
   SITHitsNonFitted = 0 ;
   TotalSITHits = 0 ;
 
-  _bField = Global::GEAR->getBField().at( gear::Vector3D(0., 0., 0.) ).z();    //The B field in z direction
+  DD4hep::Geometry::LCDD& lcdd = DD4hep::Geometry::LCDD::getInstance();
```

Comment:
Nope, '''MarlinUtil::getBzAtOrigin''' is clearly only a workaround as it uses the the singleton lcdd. Eventually this needs to be replaces with the correct lcdd object to be used in every processor...